modules/data.atmosphere/NAMESPACE

```diff
@@ -1,5 +1,8 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(extract,local.CMIP5)
+S3method(extract,local.NLDAS)
+S3method(extract,nc)
```

Comment:
This change shouldn't happen -- it's caused by Roxygen getting confused about things you didn't touch. To fix, delete your local copy of this NAMESPACE file and then regenerate it by running Roxygen from a clean session -- probably easiest from a terminal:

```
cd /path/to/your/copy/of/pecan
rm modules/data.atmosphere/NAMESPACE
make .doc/modules/data.atmosphere
```